src/app/auth/register/personal-info/personal-info.component.ts

```diff
@@ -36,9 +40,13 @@ export class PersonalInfoComponent implements OnInit, OnDestroy {
   constructor(private auth: AuthService, private register: RegisterComponent) {}
 
   ngOnInit() {
```

Comment:
If this method is not being used, it should be removed (along with the import and implement)